plugins/versionpress/src/ChangeInfos/UntrackedChangeInfo.php

```diff
@@ -26,13 +26,13 @@ public function getCommitMessage()
         return $this->commitMessage;
     }
 
-    public static function buildFromCommitMessage(CommitMessage $commitMessage)
+    public function getChangeDescription()
     {
-        return new self($commitMessage);
+        return $this->commitMessage->getUnprefixedSubject();
     }
 
-    public function getChangeDescription()
+    public function getPriority()
     {
-        return $this->commitMessage->getUnprefixedSubject();
+        return 0;
```

Comment:
`UntrackedChangeInfo` represents manual commit; thus, there are no other `ChangeInfo` objects to compare with. Plus, ignoring the fact it will not compare with anything, zero is the highest priority.